<?xml version="1.0"?>
<launch>
  <include file="$(find openni2_launch)/launch/openni2.launch">
    <arg name="depth_registration" value="true"/>
  </include>
  <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager"/>
  <node pkg="nodelet" type="nodelet" name="images2cloud" args="load depth_image_proc/point_cloud_xyzrgb nodelet_manager">
    <remap from="depth_registered/image_rect" to="camera/depth/image"/>
    <remap from="depth_registered/points" to="camera/depth_registered/points"/>
    <remap from="rgb/image_rect_color" to="camera/rgb/image_rect_color"/>
    <remap from="rgb/camera_info" to="camera/rgb/camera_info"/>
  </node>
</launch>
